### ISSUE TITLE
dark mode 

### ISSUE THREAD

**Issue by khareyash05** [navidrome/website#68]
I would like to add dark mode to the website

**Comment by deluan** [comment]
I don't think this is easily achievable without support from Docsy itself... You are free to try, but if it requires deviating too much form Docsy, I don't think we should add it

**Comment by deluan** [comment]
Maybe there's still hope for a Dark mode in a not so distant future: https://github.com/google/docsy/issues/331#issuecomment-1492027737
